README.md

```diff
@@ -1,46 +1,38 @@
 ## How to build
 
-Use `make help` to list possible tasks. But you probably want either
+Use `make help` (`gradlew tasks`) to list possible tasks. But you probably want either
 
-*  `make dist`
+*  `make dist` (`gradlew installDist`)
    which will build a runnable script for you at `cypher-shell/build/install/cypher-shell`
 
 * `make zip`
    which builds a runnable script and packages it up for you as: `out/neo4j-shell.zip`
 
 You can then just run the executable under the `bin/` sub-directory.
 
+## How to run, the fast way
 
-## Development
+This clears any previously known neo4j hosts, starts a throw-away
+instance of neo4j, and connects to it.
 
-### Integration tests
+```sh
+rm -rf ~/.neo4j/known_hosts
+docker run --detach -p 7687:7687 -e NEO4J_AUTH=none neo4j:3.0
+make run
+```
 
-Integration tests are usually skipped when you run the Gradle task `test`.
+## Development
 
-Use Gradle task `integrationTest` to specifically run integrationTests
+### Integration tests
 
-### Pre Requisites
+#### Pre Requisites for running integration tests
 
 Neo4j server with bolt driver configured.
```

Comment:
There are required if authentication is enabled, on team city they are not, and the integration tests run fine. If authentication is disabled in neo4j, then user/pass is simply ignored.